src/main/java/servlets/admin/userManagement/DowngradeAdmin.java

```diff
@@ -0,0 +1,153 @@
+package servlets.admin.userManagement;
+
+import java.io.IOException;
+import java.io.PrintWriter;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.Cookie;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpSession;
+
+import org.apache.log4j.Logger;
+import org.owasp.encoder.Encode;
+
+import utils.ShepherdLogManager;
+import utils.Validate;
+import dbProcs.Getter;
+import dbProcs.Setter;
+
+/**
+ * Control class of the Downgrade player to admin functionality
+ * <br/><br/>
+ * This file is part of the Security Shepherd Project.
+ * 
+ * The Security Shepherd project is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.<br/>
+ * 
+ * The Security Shepherd project is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.<br/>
+ * 
+ * You should have received a copy of the GNU General Public License
+ * along with the Security Shepherd project.  If not, see <http://www.gnu.org/licenses/>. 
+ *
+ */
+public class DowngradeAdmin extends HttpServlet

```

Comment:
This Servlet needs a Junit Test. You can use the examples from the other Admin Servlet JUnit Tests

Reply:
Added DowngradeAdminsTest